entity-services-examples/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,4 @@
-#Mon Oct 24 10:09:58 PDT 2016
+#Tue Nov 08 15:53:05 PST 2016
```

Comment:
no, this kind of change just happens when you run gradle wrapper.  I could have excluded it from this PR, it's a no-op.